benefit/data/sheets/FixedLists/FixedLists-posthearing-nonprod.json

```diff
@@ -14,6 +14,7 @@
   {"LiveFrom": "23/11/2022", "ID": "FL_postHearingRequestType", "ListElementCode": "statementOfReasons", "ListElement": "Statement of Reasons"},
   {"LiveFrom": "23/11/2022", "ID": "FL_requestFormat", "ListElementCode": "upload", "ListElement": "Upload request"},
   {"LiveFrom": "23/11/2022", "ID": "FL_requestFormat", "ListElementCode": "generate", "ListElement": "Enter request details"},
+  {"LiveFrom": "01/06/2023", "ID": "FL_writeStatementOfReasons", "ListElementCode": "inTime", "ListElement": "In time statement of reasons"},
```

Comment:
Is this meant to be lower case? I notice other uses are Statement of Reasons.

Reply:
This follows the descriptions in the ticket